### PR TITLE
[VEN-2142]: add forced liquidation for users VIP

### PR DESCRIPTION
**Pull request by kkirka**

### CODE REVIEW

File: simulations/vip-210/vip-210/simulations.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+forking(33715200, () => {
+  let unitroller: Contract;
+  let comptroller: Contract;
+
+  before(async () => {
+    unitroller = await ethers.getContractAt(UNITROLLER_ABI, UNITROLLER);
+    comptroller = await ethers.getContractAt(COMPTROLLER_ABI, UNITROLLER);
+  });
+
+  testVip("VIP-206 Forced liquidations for user", vip210(), {
```
Comment (chechu) [inline simulations/vip-210/vip-210/simulations.ts:33]:
```suggestion
  testVip("VIP-210 Forced liquidations for user", vip210(), {
```

File: simulations/vip-210/vip-210/simulations.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      expect(await comptroller.isForcedLiquidationEnabledForUser(SOME_USER, VUSDT)).to.be.true;
+    });
+
+    it("should enable forced liquidation via fast-track timelock", async () => {
+      const timelock = await initMainnetUser(FAST_TRACK_TIMELOCK, parseEther("1"));
+      await comptroller.connect(timelock)._setForcedLiquidationForUser(SOME_USER, VUSDT, true);
+      expect(await comptroller.isForcedLiquidationEnabledForUser(SOME_USER, VUSDT)).to.be.true;
+    });
+
+    it("should enable forced liquidation via normal timelock", async () => {
```
Comment (chechu) [inline simulations/vip-210/vip-210/simulations.ts:61]:
```suggestion
    it("should enable forced liquidation via critical timelock", async () => {
```

File: simulations/vip-210/vip-210-testnet/simulations.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+forking(35118000, () => {
+  let unitroller: Contract;
+  let comptroller: Contract;
+
+  before(async () => {
+    unitroller = await ethers.getContractAt(UNITROLLER_ABI, UNITROLLER);
+    comptroller = await ethers.getContractAt(COMPTROLLER_ABI, UNITROLLER);
+  });
+
+  testVip("VIP-206 Forced liquidations for user", vip210(), {
```
Comment (chechu) [inline simulations/vip-210/vip-210-testnet/simulations.ts:32]:
```suggestion
  testVip("VIP-210 Forced liquidations for user", vip210(), {
```

File: simulations/vip-210/vip-210-testnet/simulations.ts [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+      expect(await comptroller.isForcedLiquidationEnabledForUser(SOME_USER, VUSDT)).to.be.true;
+    });
+
+    it("should enable forced liquidation via fast-track timelock", async () => {
+      const timelock = await initMainnetUser(FAST_TRACK_TIMELOCK, parseEther("1"));
+      await comptroller.connect(timelock)._setForcedLiquidationForUser(SOME_USER, VUSDT, true);
+      expect(await comptroller.isForcedLiquidationEnabledForUser(SOME_USER, VUSDT)).to.be.true;
+    });
+
+    it("should enable forced liquidation via normal timelock", async () => {
```
Comment (chechu) [inline simulations/vip-210/vip-210-testnet/simulations.ts:55]:
```suggestion
    it("should enable forced liquidation via critical timelock", async () => {
```

**Review by chechu — APPROVED** [review]

**Review by Debugger022 — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/209

**Review by chechu — APPROVED** [review]

**Review by Debugger022 — APPROVED** [review]

**Pull request merged**